src/components/filters/MeasureValueFilter/MeasureValueFilter.tsx

```diff
@@ -0,0 +1,84 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { AFM } from "@gooddata/typings";
+
+import { MeasureValueFilterDropdown } from "./MeasureValueFilterDropdown";
+import MeasureValueFilterButton from "./MeasureValueFilterButton";
+import { IMeasureValueFilterCommonProps } from "./typings";
+
+export interface IMeasureValueFilterProps extends IMeasureValueFilterCommonProps {
+    buttonTitle: string;
+}
+
+interface IMeasureValueFilterState {
+    displayDropdown: boolean;
+}
+
+export class MeasureValueFilter extends React.PureComponent<
+    IMeasureValueFilterProps,
+    IMeasureValueFilterState
+> {
+    public state: IMeasureValueFilterState = {
+        displayDropdown: false,
+    };
+
+    private buttonRef = React.createRef<HTMLDivElement>();
+
+    public render() {
+        const { displayDropdown } = this.state;
+        const {
+            filter,
+            buttonTitle,
+            usePercentage,
+            warningMessage,
+            locale,
+            separators,
+            displayTreatNullAsZeroOption,
+            treatNullAsZeroDefaultValue,
+        } = this.props;
+
+        return (
+            <>
```

Comment:
I saw some examples are using <React.Fragment> and this code just <>. Probably examples are just reworked previous ones? Could you please make them consistent?